PIM/Integration/01_ManageAttributes.md

```diff
@@ -0,0 +1,213 @@
+
+3. Select a data type in the *Data type* drop-down list.   
+ The *CONFIGURATION* section is displayed. For detailed information about all data types, see [Data type list](../UserInterface/04_DataTypeList.md).
+
+   > [Info] The settings displayed in the *CONFIGURATION* section depend on the selected data type.
+
+4. Enter a key for the attribute in the *Key* field. The key is required for API access and must be system wide unique.
+
+  > [Info] In order to facilitate the assignment of attributes in the further process (for instance in the ETL mapping), it is recommended to add the prefix **pim_** to all attributes created in the *PIM* module.     
+
+5. If desired, enable the *Multi-language* toggle to assign values in multiple languages to the attribute or enable the *Multi-channel* toggle to assign different attribute values in different channels.
+
+  > [Info] Be aware that only attributes which are neither multi-language nor multi-channel can be used as defining attributes for variants. For detailed information, see [Create a variant set](./07_ManageVariantSets.md#create-a-variant-set).
+
+6. Click the ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button in the *Assigned Sets* field. The button is locked if you have not yet selected a data type.   
+  A drop-down list with all active attribute sets is displayed.
+
+7. Select an attribute set in the *Assigned Sets* drop-down list.
+
+  > [Info] You can assign the attribute to multiple sets. Repeat the steps **7** to **8** to assign the attribute to a further attribute set. To delete the assignment to a selected set, click the ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete) button right to the set.
+
+8. If required, configure the settings in the *CONFIGURATION* section.   
+  For detailed information about the different configuration settings, see [Data type list](../UserInterface/04_DataTypeList.md).
+
+  > [Info] By default, the status of a new attribute is set to active. To deactivate a new attribute for use, disable the *Active* toggle. For detailed information, see [Deactivate an attribute](#deactivate-an-attribute).
+
+9. Click the [SAVE] button in the upper right corner.   
+  The new attribute has been saved. The *Create Attribute* view is closed.  
+
+
+
+## Edit an attribute
+
+After you have created an attribute, you can edit it. However, only some attribute properties are editable. For instance, the data type and the attribute key cannot be subsequently modified. The assignment of an attribute to an attribute set must be changed in the [attribute set](ManageAttributeSet.md) itself.   
+There are also some attributes that are automatically created by the system, for instance when installing a plugin or module. These system attributes are read-only and cannot be edited. It is highly recommended not to deactivate these attributes to avoid interfering with the functioning of the *Core1 Platform*.
+
```

Comment:
Ich finde, diese Infos sind so schwierig zu verstehen. Vielleicht etwas strukturierter? Z.B.: 
- Properties that are not editable: data type and key 
- Properties that are edited somewhere else (in Attribute set)
- Properties automatically created by the system and there read-only.

Reply:
Ich lese gerade die Info unter dem Screenshot. Vielleicht irgendwie beide Infos zusammenfügen?